### PR TITLE
Adding email wrapping lib

### PR DESCRIPTION
**Pull request by akeem**
# Description

Adding email wrapping lib

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: nudge/tests/email.test.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    .mockReturnValueOnce({ statusCode: 204 })
+    .mockReturnValueOnce({ statusCode: 500 })
+})
+
+describe('Email', () => {
+  describe('dispatch', () => {
+    describe('when the email request was accepted', () => {
+      it('returns true', async () => {
+        expect.assertions(1)
+        let successfulDispath = await Email.dispatch({
```
Comment (cnasc) [inline nudge/tests/email.test.ts]:
```suggestion
        let successfulDispatch = await Email.dispatch({
```

File: nudge/tests/email.test.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        })
+
+        expect(successfulDispath).toBe(true)
+      })
+    })
+
+    describe('when there is an issue with the email request', () => {
+      it('returns false', async () => {
+        expect.assertions(1)
+        let unsuccessfulDispath = await Email.dispatch({
```
Comment (cnasc) [inline nudge/tests/email.test.ts]:
```suggestion
        let unsuccessfulDispatch = await Email.dispatch({
```

**Comment by akeem** [comment]
@cnasc otra vez, por favor

File: nudge/src/email.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv'
+import request from 'request-promise-native'
+
+dotenv.config()
```
Comment (julien51) [inline nudge/src/email.ts]: Do you want to use the same config loading stuff used in other places?

Comment (akeem) [reply nudge/src/email.ts]: nah, I think this is fine for now. It will run in a dockerized environment with the configuration explicitly passed to it

**Pull request merged**